libc-test/build.rs

```diff
@@ -3574,19 +3569,16 @@ fn test_vxworks(target: &str) {
         _ => false,
     });
     // FIXME(vxworks)
-    cfg.skip_type(move |ty| match ty {
-        "stat64" | "sighandler_t" | "off64_t" => true,
-        _ => false,
-    });
+    cfg.skip_type(|ty| matches!(ty, "stat64" | "sighandler_t" | "off64_t"));
 
-    cfg.skip_field_type(move |struct_, field| match (struct_, field) {
-        ("siginfo_t", "si_value") | ("stat", "st_size") | ("sigaction", "sa_u") => true,
-        _ => false,
+    cfg.skip_field_type(|struct_, field| {
+        matches!(
+            (struct_, field),
+            ("siginfo_t", "si_value") | ("stat", "st_size") | ("sigaction", "sa_u")
+        )
```

Comment:
Could you drop the changes to `matches!`  here? It's easier and less conflict-y to append new types/fields in the current form, plus consistency.